rust/syntect-plugin/examples/make_manifest.rs

```diff
@@ -21,12 +21,14 @@ use std::fs::{self, File};
 use std::io::{self, Write};
 use std::path::{Path, PathBuf};
 
+use crate::xi_core::config::ConfigManager;
 use crate::xi_core::plugin_manifest::*;
 use crate::xi_core::LanguageDefinition;
+use syntect::dumps::dump_to_file;
 use syntect::parsing::{SyntaxReference, SyntaxSet};
 use toml::Value;
 
-const OUT_FILE_NAME: &str = "generated_manifest.toml";
+const OUT_FILE_NAME: &str = "manifest.toml";
```

Comment:
hmm, I remember having _some_ reason why I wanted to use a different file name, but I forget what it was. 😕

Reply:
The xi-editor ignores other files. I will have to write my changes on to `manifest.toml` else the changes aren't processed by the editor.

Reply:
Yea, I think I did it the other way so that I could diff the two files, and then I would manually replace `manifest.toml` when I was satisfied? This doesn't seem necessary, though.